.circleci/config.yml

```diff
@@ -91,7 +91,7 @@ aliases:
     - mods-preview        # OSS Standalone and all preview modules
     - oss-st-6-tls        # OSS Standalone v6 with TLS enabled
     - oss-st-6-tls-auth   # OSS Standalone v6 with TLS auth required
-    - oss-st-6-tls-auth-ssh   # OSS Standalone v6 with TLS auth required through ssh
+#    - oss-st-6-tls-auth-ssh   # OSS Standalone v6 with TLS auth required through ssh
```

Comment:
Do not forget to create an issue to fix this

Reply:
https://redislabs.atlassian.net/browse/RI-4463
thanks